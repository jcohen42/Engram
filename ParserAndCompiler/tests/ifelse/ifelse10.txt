Begin program:
	Set x equal to 76.
	Set y equal to 343.
	If x is not equal to y:
		Output "Not equal".
		If x is less than y:
			Output "x less than y".
			If y is greater than x:
				Output "y greater than x".
			End if.
		End if.
	End if.
End program.