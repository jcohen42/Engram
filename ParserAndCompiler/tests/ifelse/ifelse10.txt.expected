Not equal
x less than y
y greater than x